types/tar/index.d.ts

```diff
@@ -198,14 +198,22 @@ export const knownExtended: {
 export const headerSize: number;
 export const blockSize: number;
 
+/**
+ * A writable stream. Write tar data to it and it will emit entry events for each entry parsed from the tarball. This is used by tar.Extract.
+ */
+export interface Parse extends ParseStream {
```

Comment:
You should be able to just export this as a class rather than a separate const and interface.

Reply:
@connor4312
Thank you! I also would like to merge this but can't `class` extends `interface` and should we list all properties of `ParseStream` to `implements` `ParseStream`?

Reply:
@connor4312 
Hi, could you please advise me about how to declare `class` without listing all properties or approve this PR?
This PR seems not perfect, but fixes our broken usage path.